### PR TITLE
Change secrets to vars, update Readme Files

### PR DESCRIPTION
**Pull request by crochcunill**
I have tested the changes by destroying an existing containers app and redeploying again (check https://ds0j219k20ylk.cloudfront.net/ if you are curious) in a fork from this repo.

The readme files have been updated, the workflows are now reading only vars instead of secrets and I have deleted three stale workflows that I created 
- Testencrypt4.yml
- encrypt4.yml

Not sure why, but some of the GitHub actions have different _uses_ versions, for example the new push_manual.yaml (line 68) is 

` uses: docker/build-push-action@v4`

instead of 

` uses: docker/build-push-action@v2`

Not sure when this change happen, but again, I was able to deploy (in my fork) with no trouble

If approved, there is another change that I will need to do manually: remove the existing secrets in this repo and replace them with variables, like https://github.com/bcgov/startup-sample-project-aws-containers-o20ya1/settings/variables/actions

### CODE REVIEW

File: .github/workflows/pr-sandbox.yaml [hunk 1]
```diff
@@ -1,11 +1,7 @@
 name: Pull Request Sandbox
 # change to trigger a PR
 on:
-  pull_request:
-    paths:
-      - ".github/workflows/pr-sandbox.yaml"
-      - ".github/workflows/tf-plan-sandbox.yaml"
-      - "terraform/sandbox/**"
+  workflow_dispatch:
```
Comment (wrnu) [inline .github/workflows/pr-sandbox.yaml:4]: This shouldn't be changing, this makes it a manual workflow and not triggered on PR

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -2,19 +2,10 @@ name: Pull Request
...
-    paths:
-      - ".github/workflows/pr.yaml"
-      - ".github/workflows/tf-plan.yaml"
-      - "client/**"
-      - "server/**"
-      - "terraform/**"
-      - "!terraform/sandbox/**"
-      - "Dockerfile"
-      - "!**/README.md"
+  workflow_dispatch:
```
Comment (wrnu) [inline .github/workflows/pr.yaml:5]: This shouldn't be changing, this makes it a manual workflow and not triggered on PR

File: .github/workflows/deploy-test.yaml [hunk 3]
```diff
@@ -1,9 +1,7 @@
 name: Deploy Test
 
 on:
-  push:
-    tags:
-      - "v[0-9].[0-9]+.[0-9]+-rc*"
+  workflow_dispatch:
```
Comment (wrnu) [inline .github/workflows/deploy-test.yaml:4]: This shouldn't be changing, this makes it a manual workflow and not triggered on tag

File: .github/workflows/deploy-prod.yaml [hunk 4]
```diff
@@ -1,13 +1,11 @@
 name: Deploy Prod
 
 on:
-  push:
-    tags:
-      - "v[0-9].[0-9]+.[0-9]+"
+  workflow_dispatch:
```
Comment (wrnu) [inline .github/workflows/deploy-prod.yaml:4]: This shouldn't be changing, this makes it a manual workflow and not triggered on tags

**Review by wrnu — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-sandbox.yaml (hunk 1 above)
Comment (crochcunill) [reply .github/workflows/pr-sandbox.yaml:4]: Done.

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (crochcunill) [reply .github/workflows/pr.yaml:5]: Done.

File: .github/workflows/deploy-prod.yaml (hunk 4 above)
Comment (crochcunill) [reply .github/workflows/deploy-prod.yaml:4]: Correction done

**Pull request closed without merging**